src/applications/appeals/995/content/itfWrapper.jsx

```diff
@@ -45,23 +45,57 @@ export const itfExpander = (
 
 export const itfError = (
   <div>
-    <p className="vads-u-font-size--base">
-      We’re sorry. Your Intent to File request didn’t go through because
-      something went wrong on our end. For help creating an Intent to File a
-      Claim for Compensation, please call Veterans Benefits Assistance at{' '}
-      <va-telephone contact={CONTACTS.VA_BENEFITS} />, Monday through Friday,
-      8:00 a.m. to 9:00 p.m. ET. Or, you can fill out VA Form 21-0966 and submit
-      it to:
-    </p>
-    <p className="va-address-block vads-u-font-size--base">
-      Department of Veterans Affairs
-      <br role="presentation" />
-      Claims Intake Center
-      <br role="presentation" />
-      PO Box 4444
-      <br role="presentation" />
-      Janesville, WI 53547-4444
-    </p>
+    <div className="vads-u-margin-bottom--2">
+      <p className="vads-u-font-size--base">
+        You can continue to file your claim or call us to confirm.
+      </p>
+      <strong>What an intent to file means: </strong>
+      <p className="vads-u-font-size--base">
+        <ul>
+          <li>
+            If you’re filing a Supplemental Claim to request a review of a claim
+            we decided longer than 1 year ago, an intent to file sets a
+            potential start date (or effective date) for your benefits. You then
+            have up to 1 year to complete and file your Supplemental Claim. And
+            you may be able to get payments for the time between when you
+            submitted your intent to file and when we approved your claim.
+          </li>
+          <li>
+            If you’re submitting a Supplemental Claim to request a review of a
+            claim we decided within the past year, an intent to file gives you 1
+            year to complete and file your Supplemental Claim. But we’ll use the
+            original date you filed the claim under review as the potential
+            start date for your benefits.
+          </li>
+        </ul>
+      </p>
+      <br />
+      <strong>What you can do next:</strong>
+      <p className="vads-u-font-size--base">
+        <ul>
+          <li>
+            You can continue and submit your claim online today—or when you’re
+            ready. If we don’t have an intent to file on record for this claim
+            that’s within the past year, we’ll use the date you submit your
```

Comment:
Looks correct here, but not in the included screenshot, where "that's" appears as "that's that's".